examples/cpp/EventSender/EventSender.cpp

```diff
@@ -71,6 +72,38 @@ const char* defaultConfig = static_cast<const char *> JSON_CONFIG
     }
 );
 
+// Mock function that performs random selection of destination URL. 1DS SDK does not define how the app needs to perform
+// the region determination. Products should use MSGraph API, OCPS, or other remote config provisioning sources, such as
+// ECS: https://learn.microsoft.com/en-us/deployedge/edge-configuration-and-experiments - in order to identify what 1DS
+// collector to use for specific Enterprise or Consumer end-user telemetry uploads. Note that the EUDB URL determination
+// is performed asynchronously and could take a few seconds. EUDB URL for Enterprise applications may be cached
+// in app-specific configuration storage. 1DS SDK does not provide a feature to cache the data collection URL used for
+// a previous session.
+std::string GetEudbCollectorUrl()
+{
+    const auto randSeed = std::chrono::duration_cast<std::chrono::milliseconds>(std::chrono::system_clock::now().time_since_epoch()).count();
+    srand(static_cast<unsigned>(randSeed));
```

Comment:
>srand [](http://example.com/codeflow?start=4&length=5)

Call `srand` once instead of every time when`GetEudbCollectorUrl` is called?

Reply:
Technically it's called once only. This is just a way to randomize the URL selection. Products must implement the actual logic to deterministically obtain the correct URL.

Reply:
I added a code comment to explain why / when.